fix(curriculum): Made tests not accept invalid HTML (#35774)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/add-images-to-your-website.english.md

```yml
tests:
  - text: Your page should have an image element.
    testString: assert($("img").length);
  - text: Your image should have a <code>src</code> attribute that points to the kitten image.
    testString: assert(/^https:\/\/bit\.ly\/fcc-relaxing-cat$/i.test($("img").attr("src")));

>   - text: Your image element's <code>alt</code> attribute <strong>must</strong> not be empty.

Reviewer: ```suggestion
  - text: Your image element's <code>alt</code> attribute must not be empty.
```
I think we just make it this since the tests come out looking like this:
![image](https://user-images.githubusercontent.com/34807532/55786140-53aca980-5ad1-11e9-93c3-ffdbc12c4f59.png)
See the `<strong>` tag displaying as is

Reviewer: @RandellDawson I fiddled around with some things and have found a way to work this. Have created a PR to fix this

>   - text: Your image element <strong>must</strong> have an <code>alt</code> attribute with a non-empty value.

Reviewer: ```suggestion
  - text: Your image element's <code>alt</code> attribute <strong>must</strong> not be empty.
```
Since we're fiddling with this line on the PR, I guess this is a better way to say what we're trying to
